fuzz/corpus/segmentaa

```diff
@@ -0,0 +1 @@
+$GPRMC,170058.89,V,3554.928,N,08002.496,W,9.4,1.57,050521,,E*41
```

Comment:
Is it necessary to commit all of these seed files? Will one or two be sufficient?